src/backend/code.d

```diff
@@ -34,6 +38,68 @@ struct LocalSection
 
 extern __gshared LocalSection Para;
 
+alias IDXSTR = uint;
+alias IDXSEC = uint;
+alias IDXSYM = uint;
+
+struct seg_data
+{
+    segidx_t             SDseg;         // index into SegData[]
+    targ_size_t          SDoffset;      // starting offset for data
+    int                  SDalignment;   // power of 2
+
+    version (Windows) // OMFOBJ
+    {
+        bool isfarseg;
+        int segidx;                     // internal object file segment number
+        int lnameidx;                   // lname idx of segment name
+        int classidx;                   // lname idx of class name
+        uint attr;                      // segment attribute
+        targ_size_t origsize;           // original size
+        long seek;                      // seek position in output file
+        void* ledata;                   // (Ledatarec) current one we're filling in
+    }
+
+    static if (1) //ELFOBJ || MACHOBJ
```

Comment:
So... 1 or a condition? Just delete this for now and insert later if needed. We need to clean such petrified lava.